airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -614,6 +614,20 @@ class TicketMetricEvents(CursorPaginationZendeskSupportStream):
     def path(self, **kwargs):
         return "incremental/ticket_metric_events"
 
+    def request_params(
```

Comment:
@artem1205 ideally we would have some test coverage for this. Is there a reason it's not covered by integration tests?

Reply:
All those stream use the same pagination mechanism implemented in `CursorPaginationZendeskSupportStream`, and raise Error if start_date **IS IN** the request_params. But yep, i'll make another test for this particular stream.

Reply:
Okay thanks. It seems that there probably wasn't test coverage for the other streams too, since there wasn't a test failure that alerted us to the issue before the code was merged. So I think it's definitely worthwhile to add it.